### PR TITLE
TH-1045 | Image credits

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Add image credits to banner heros on landing page
- Add image credits to collection page hero

## Additional notes :spiral_notepad:

### CODE REVIEW

**Review by melniiv — APPROVED** [review]

File: src/domain/banner/bannerHero/BannerHero.tsx [hunk 1]
```diff
@@ -135,6 +138,20 @@ const BannerHero: React.FC<Props> = ({ banner, location }) => {
           }}
         />
       )}
+      {heroImageCredits && (
+        <div
+          className={classNames(
+            styles.imageCredits,
+            titleAndDescriptionColor &&
```
Comment (nenonja) [inline src/domain/banner/bannerHero/BannerHero.tsx:145]: This is repeated twice, could be a const
`const titleAndDescriptionStyle= titleAndDescriptionColor &&             styles[`color${capitalize(titleAndDescriptionColor)}`]`

Comment (nenonja) [inline src/domain/banner/bannerHero/BannerHero.tsx:145]: This is repeated twice, could be a const
`const titleAndDescriptionStyle= titleAndDescriptionColor && styles[color${capitalize(titleAndDescriptionColor)}]`

File: src/domain/banner/bannerHero/BannerHero.tsx [hunk 2]
```diff
@@ -46,10 +47,16 @@ export const getTestIds = (
   heroTopLayerImage: `${location}-heroTopLayerImage`,
 });
 
+const contentBackgroundColorMap: Record<LandingPageTextColor, string> = {
```
Comment (nenonja) [inline src/domain/banner/bannerHero/BannerHero.tsx:50]: This and some functions above could be in `bannerUtils.ts`

File: src/domain/banner/bannerHero/__tests__/BannerHero.test.tsx [hunk 3]
```diff
@@ -4,23 +4,43 @@ import { axe } from 'jest-axe';
...
+    />
+  );
 
   expect(screen.getByText(title)).toBeInTheDocument();
   expect(screen.getByText(description)).toBeInTheDocument();
   expect(screen.getByText(buttonText)).toBeInTheDocument();
+
+  const creditsContainer = screen.queryByText(`Kuva: ${photographerCredit}`);
+  expect(creditsContainer).toBeInTheDocument();
+  expect(creditsContainer).toHaveStyle('background-color: rgba(0, 0, 0, 0.7)');
```
Comment (nenonja) [inline src/domain/banner/bannerHero/__tests__/BannerHero.test.tsx:43]: `.toHaveStyle('background-color: ${contentBackgroundColorMap.BLACK}');`

**Review by nenonja — COMMENTED** [review]
Some minor comments... They can be either fixed or ignored :) How do I test this?

**Comment by nenonja** [comment]
Something is broken either in api-proxy or ui. Credit data is present only through finnish locale

**Pull request merged**